components/LobbyCircle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from '../styles/LobbyCircle.module.scss';
+
+enum CreateOptions {
+  NotCreate,
+  CreateOnly,
+  CreateWithLobbies
+}
+
+export interface LobbyCircleProps {
+  name: string;
+  create?: CreateOptions;
+}
+
+export function LobbyCircle({name, create = CreateOptions.NotCreate}: LobbyCircleProps): JSX.Element {
+  const CreateOptions2Style = {
+    [CreateOptions.CreateOnly]: styles.single,
+    [CreateOptions.CreateWithLobbies]: styles.create,
+  };
+
+  const additionalStyle = create ? CreateOptions2Style[create] : styles.top;
```

Comment:
i lied this doesnt work because you have `CreateOptions.NotCreate` lol I think the fix is to remove all of this and just do smth like

```
const additionalStyle = CreateOptions2Style[create];
```